ui/tests/helpers/clients/client-count-helpers.js

```diff
@@ -646,10 +844,369 @@ export const SERIALIZED_ACTIVITY_RESPONSE = {
+                entity_clients: 0,
+                non_entity_clients: 0,
+                secret_syncs: 100,
+              },
+            },
+          },
+        },
+      },
+      new_clients: {
+        month: '7/23',
+        timestamp: '2023-07-01T00:00:00Z',
+        acme_clients: 100,
+        clients: 100,
+        entity_clients: 100,
+        non_entity_clients: 100,
+        secret_syncs: 100,
+        namespaces: [
+          {
+            label: 'root',
+            acme_clients: 100,
+            clients: 100,
+            entity_clients: 100,
+            non_entity_clients: 100,
+            secret_syncs: 100,
+            mounts: [
+              {
+                label: 'pki-engine-0',
+                acme_clients: 100,
+                clients: 100,
+                entity_clients: 0,
+                non_entity_clients: 0,
+                secret_syncs: 0,
+              },
+              {
+                label: 'auth/authid/0',
+                acme_clients: 0,
+                clients: 100,
+                entity_clients: 100,
+                non_entity_clients: 100,
+                secret_syncs: 0,
+              },
+              {
+                label: 'kvv2-engine-0',
+                acme_clients: 0,
+                clients: 100,
+                entity_clients: 0,
+                non_entity_clients: 0,
+                secret_syncs: 100,
+              },
+            ],
+          },
+        ],
+      },
+    },
+    {
+      month: '8/23',
+      timestamp: '2023-08-01T00:00:00Z',
+      acme_clients: 100,
+      clients: 100,
+      entity_clients: 100,
+      non_entity_clients: 100,
+      secret_syncs: 100,
+      namespaces: [
+        {
+          label: 'root',
+          acme_clients: 100,
+          clients: 100,
+          entity_clients: 100,
+          non_entity_clients: 100,
+          secret_syncs: 100,
+          mounts: [
+            {
+              label: 'pki-engine-0',
+              acme_clients: 100,
+              clients: 100,
+              entity_clients: 0,
+              non_entity_clients: 0,
+              secret_syncs: 0,
+            },
+            {
+              label: 'auth/authid/0',
+              acme_clients: 0,
+              clients: 100,
+              entity_clients: 100,
+              non_entity_clients: 748,
+              secret_syncs: 0,
+            },
+            {
+              label: 'kvv2-engine-0',
+              acme_clients: 0,
+              clients: 100,
+              entity_clients: 0,
+              non_entity_clients: 0,
+              secret_syncs: 100,
+            },
+          ],
+        },
+      ],
+      namespaces_by_key: {
+        root: {
+          acme_clients: 100,
+          clients: 100,
+          entity_clients: 100,
+          non_entity_clients: 100,
+          secret_syncs: 100,
+          timestamp: '2023-08-01T00:00:00Z',
+          month: '8/23',
+          // TODO: this isn't right
```

Comment:
Can this `new_clients` be updated to what we expect it to look like after serialization?

Reply:
thanks, exactly why I added the todo 😅